<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>view_schedule_window</class>
 <widget class="QMainWindow" name="view_schedule_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Checkout</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="schedule_table_widget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>811</width>
      <height>371</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Course ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Course Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Section</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Instructor</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Days</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Start Time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>End Time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Room</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="schedule_label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>20</pointsize>
      <bold>true</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Checkout</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enroll_button">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>550</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Enroll</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
